rest-api/src/main/java/com/cloudbees/workflow/rest/external/ChangeSetExt.java

```diff
@@ -45,6 +45,16 @@
     private List<Commit> commits;
     private String consoleUrl; // Not a rest endpoint so not including in _links
 
+
+    /** Allows user to disable Jenkins user lookup for commit authors
+     * By setting System Property com.cloudbees.workflow.rest.external.ChangeSetExt.resolveCommitAuthors to 'false'
+     * This is a workaround for JENKINS-35484 where user lookup encounters issues */
+    private static boolean resolveCommitAuthors() {
+        String prop = System.getProperty(ChangeSetExt.class.getName()+".resolveCommitAuthors");
+        // Anything but something *explicitly* matching "false" (case insensitive) needs to return true
+        return (prop == null || prop.isEmpty() || !"false".equalsIgnoreCase(prop));
```

Comment:
@jglick Intentional.  The reason for the comment here: https://github.com/jenkinsci/pipeline-stage-view-plugin/pull/15#discussion-diff-66810938R54 is that Boolean.parseBoolean will return false for anything that doesn't match 'true' (case insensitive).  

We want the opposite, for safety: resolve commit authors unless they explicitly set the system property to 'false' (case insensitive).  That means we don't disable user lookup if they've mangled the setting (avoids some possible issues).